File: JavaScript/8js 3/mouthSize.js
/*
|--------------------------------------------------------------------------
| The Wide-Mouthed frog!
|--------------------------------------------------------------------------
|
*/

function mouthSize(animal) {
   return animal.toLowerCase() == "alligator" ? "small" : "wide";
}


function mouthSize(animal) {
   return animal.match(/alligator/i) ? 'small' : 'wide';
}


function mouthSize(animal) {
   return /alligator/gi.test(animal) ? 'small' : 'wide'
}


function mouthSize(animal) {
   return animal[1].toLowerCase() == "l" ? "small" : "wide"
}


console.log(mouthSize("toucan"), "wide")
console.log(mouthSize("ant bear"), "wide")
console.log(mouthSize("alligator"), "small")

File: Python/8py 3/mouth_size.py
"""
|--------------------------------------------------------------------------
| The Wide-Mouthed frog!
|--------------------------------------------------------------------------
|
"""

def mouth_size(animal): 
   return "small" if animal.lower() == "alligator" else "wide"


def mouth_size(animal): 
   return ["wide","small"][animal.lower()=="alligator"]


def mouth_size(animal):
   return "swmiadlel"[animal.lower() != "alligator"::2]


print(mouth_size("toucan"),"wide")
print(mouth_size("ant bear"),"wide")
print(mouth_size("alligator"),"small")